### PR TITLE
style: Set Docker memory and CPU limits

### PR DESCRIPTION
**Pull request by kare**

### CODE REVIEW

**Comment by kare** [comment]
@samt1803 @timoxley Do these limits seem reasonable to you guys?

File: docker-compose.yml [hunk 1]
```diff
@@ -246,6 +351,13 @@ services:
           interval: 10s
           timeout: 10s
           retries: 60
+        deploy:
+            resources:
+                limits:
+                    cpus: '0.50'
+                    memory: 50M
```
Comment (timoxley) [inline docker-compose.yml:358]: Hm, what does the `platform` container actually run? it probably runs a webserver right, in which case this is probably rather low, especially if it has to build the webpack bundle. cc @juhah @tumppi @mondoreale

**Review by timoxley — COMMENTED** [review]
The 100MB limits seem rather low for node processes, but might be worth running with these low limits and only bumping them up when something/someone complains.

**Review by samt1803 — APPROVED** [review]
The trackers, brokers and ipfs go to about 70, to 80% of their RAM, other than that they all stay well below their limits. 
(If something does not work, like some databse being offline, many containers like the bridge related ones hit their limits, but that might be a good thing and then needs to be fixed by resolving the underlying issue)

File: docker-compose.yml (hunk 1 above)
Comment (tumppi) [reply docker-compose.yml:358]: `platform` runs nginx serving prebuilt app bundle so no webpack is needed and should not need much resources.

`network-explorer` on the other hand uses webpack dev server currently so it needs more resources. We can get rid of webpack and use nginx instead when we have enough runway to implement this.

**Pull request merged**